docs/docs/guide/debugging_table/details_map.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  chromeDebuggerHermesAndroid,
+  chromeDebuggerHermesiOS,
+  chromeDebuggerJSCAndroid,
+  chromeDebuggerJSCiOS,
+  chromeDebuggerV8Android,
+  chromeDebuggerV8iOS,
+  chromeDevToolsHermesAndroid,
+  chromeDevToolsHermesiOS,
+  chromeDevToolsJSCAndroid,
+  chromeDevToolsJSCiOS,
+  chromeDevToolsV8Android,
+  chromeDevToolsV8iOS,
+  flipperHermesAndroid,
+  flipperHermesiOS,
+  flipperJSCAndroid,
+  flipperJSCiOS,
+  flipperV8Android,
+  flipperV8iOS,
+  nothingSelected,
+  reactDevToolsHermesAndroid,
+  reactDevToolsHermesiOS,
+  reactDevToolsJSCAndroid,
+  reactDevToolsJSCiOS,
+  reactDevToolsV8Android,
+  reactDevToolsV8iOS,
+  safariDevToolsHermesAndroid,
+  safariDevToolsHermesiOS,
+  safariDevToolsJSCAndroid,
+  safariDevToolsJSCiOS,
+  safariDevToolsV8Android,
+  safariDevToolsV8iOS,
+} from './details';
+
+export const detailsMap = new Map<string, JSX.Element>([
+  ['', nothingSelected()],
+  ['Chrome Debugger/JSC/Android', chromeDebuggerJSCAndroid()],
```

Comment:
I wonder if this huge details map can be converted into a function (or even React component) that accepts the configuration as parameters, e.g.:
```ts
function Details({ tool, vm, platform }) {
  if (/* ... */) {
    return <ChromeDebuggerJSCAndroid />;
  }
  // ...
}
```
This way we can not only compress the logic but also automatically fill table cells.